biz.aQute.bnd.annotation/src/aQute/bnd/annotation/spi/Constants.java

```diff
@@ -0,0 +1,18 @@
+package aQute.bnd.annotation.spi;
+
+public interface Constants {
```

Comment:
I prefer we use a class (not an interface) for constant definition. See Effective Java, 2nd Ed `#19`.